src/fragments/lib/datastore/js/relational/query-many-snippet.mdx

```diff
@@ -1,48 +1,48 @@
-These many-to-many relationship models can be queried directly to return all models in the relationship:
+### Create
+
+In order to save a many-to-many relationship, create both model instance first and then save them to a new join model instance.
 
 ```js
-// All post editor relationships
-const results = await DataStore.query(PostEditor);
+const post = await DataStore.save(new Post({
+  title: 'Amplify Weekly'
+  rating: 10,
+  status: PostStatus.ACTIVE,
+}))
+
+const user = await DataStore.save(new User({
+  username: 'rene'
+}))
+
+const postEditor = await DataStore.save(new PostEditor({
+  post: post,
+  user: user
+}))
 ```
 
-This will return an array of `PostEditor`s with `Post` and `Editor` model instances attached. For example, without metadata, `results` from above would look like this:
-
-```json
-[
-    {
-        "id": "4b66cee3-1436-4d53-910f-7cfe0d955cd8",
-        "post": {
-            "id": "d2a96183-938f-4469-9873-944336fb9d9d",
-            "title": "My first post"
-        },
-        "editor": {
-            "id": "2cbfdd83-8353-4b0e-ae63-8f7d004c728f",
-            "username": "Nadia"
-        }
-    }
-]
+Here we've first created a new `Post` instance and a new `User` instance. Then, saved those instances to a new instance of our join model `PostEditor`.
+
+### Query
+
+To query many-to-many relationships, query either model with a nested predicate of the other model's primary key.
```

Comment:
I think the other model's here is a big unclear.

Reply:
do you have a suggestion?

Reply:
```suggestion
To query many-to-many relationships, use a nested predicate filter the target model by the source model's id:
```